src/clear/extensions/time/time_in_day.cr

```diff
@@ -0,0 +1,139 @@
+# Represents the "time" object of PostgreSQL
+struct TimeInDay
+  getter ms : UInt64 = 0
+
+  SECOND = 1_000_000_u64
+  MINUTE = 60_u64 * SECOND
+  HOUR = 60_u64 * MINUTE
+
+  def initialize(hours, minutes, seconds = 0)
+    @ms = SECOND * seconds + MINUTE * minutes + HOUR * hours
```

Comment:
Does crystal support order of operations?
It might be easier to. Understand if these are in parenthesis.

```suggestion
    @ms = (SECOND * seconds) + (MINUTE * minutes) + (HOUR * hours)
```

Reply:
Yes, Crystal support order of operations.

I'm quite neutral about the parenthesis.

Reply:
that is awesome to know.  I guess this is not as big of a deal once you know that.